q2_types/per_sample_sequences/tests/test_transformer.py

```diff
@@ -198,7 +206,7 @@ def test_miseq_demux_dirfmt_to_slpspefdf(self):
     def test_fastqmanifest_single(self):
         _, dirfmt = self.transform_format(
             CasavaOneEightSingleLanePerSampleDirFmt,
-            SingleLanePerSamplePairedEndFastqDirFmt,
+            SingleLanePerSampleSingleEndFastqDirFmt,
```

Comment:
Note, this appears to be the only "real" error in this test suite, so if we decide to close this PR let's add a TODO/issue to fix this little typo.